tests/test_hetero_random_walk.cc

```diff
@@ -16,6 +16,10 @@ static bool check_if_edge_exist(int64_t src, int64_t dst, int64_t *indptr_arr, i
     }
     int64_t begin = indptr_arr[src];
     int64_t end = indptr_arr[src + 1];
+    std::cout << "begin:" << begin << std::endl;
```

Comment:
Remove all std::cout and unnecessary comments.

Reply:
OK, unnecessary comments are removed